src/server/twitterSearch.js

```diff
@@ -15,6 +15,16 @@ var T = new Twit({
   'timeout_ms':           60 * 1000,  // optional HTTP request timeout to apply to all requests.
 });
 
+const makeTweetFromRaw = (raw) => {
+  return Builders.TweetBuilder()
+    .id(raw.id)
+    .content(raw.text)
+    .date(moment(new Date(raw.created_at)).format('YYYY-MM-DD HH:mm:ss'))
+    .likes(raw.favourite_count || 0)
+    .retweets(raw.retweet_count || 0)

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'retweet_count' is not in camel case.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2175855561/issues/source?bid=3148782&fileBranchId=3163167#l24)